src/main/java/taxi/model/Driver.java

```diff
@@ -50,11 +76,22 @@ public boolean equals(Object o) {
         Driver driver = (Driver) o;
         return Objects.equals(id, driver.id)
                 && Objects.equals(name, driver.name)
-                && Objects.equals(licenseNumber, driver.licenseNumber);
+                && Objects.equals(licenseNumber, driver.licenseNumber)
+                && Objects.equals(login, driver.login)
+                && Objects.equals(password, driver.password);
```

Comment:
Please do not use confidential data anywhere outside absolutely necessary for authentication